docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -72,10 +72,10 @@
     from lightly.utils.cropping.crop_image_by_bounding_boxes import crop_dataset_by_bounding_boxes_and_save
 
     BASE_PATH = "path/to/dataset/"
-    DATASET_PATH = BASE_PATH+"images"  # the path were the full images are found
-    OUTPUT_DIR = BASE_PATH+"cropped_images" # the path where the cropped images will be saved
+    DATASET_PATH = os.path.join(BASE_PATH,"images")  # the path were the full images are found
```

Comment:
Sorry for being so picky but you should always have a space after a comma.

Reply:
That's write.
Usually my IDE tells me this, but it does not know that this is python code.